Makefile

```diff
@@ -12,8 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+DEFAULT_ARCH = amd64
```

Comment:
The difference between DEFAULT_ARCH and ARCH seems unclear.

As an alternative, just a new `RELEASE_LIST = amd64`?